comixed-webui/src/assets/i18n/en/admin.json

```diff
@@ -21,7 +21,7 @@
       "renaming-rule-format": "The comic format",
       "renaming-rule-issue-number": "The issue number",
       "renaming-rule-published-month": "The month published",
-      "renaming-rule-published-year": "The name of the publisher",
+      "renaming-rule-published-year": "The year published",
```

Comment:
Thanks for catching that! To keep the other translations consistent, would you mind adding this same change to the other admin.json files? That way other translators will see what was intended.